src/open_inwoner/accounts/views/auth.py

```diff
@@ -139,3 +144,86 @@ def get_success_url(self):
             del session["invite_url"]
 
         return super().get_success_url()
+
+
+class CustomeHerkenningAssertionConsumerServiceMockView(
+    eHerkenningAssertionConsumerServiceMockView
+):
+    def get_login_url(self):
```

Comment:
Could be `get_fail_url` or sth. similar, as in a previous comment.

Reply:
I modified this to be `get_failure_url` in the mock views in `src/eherkenning`. Because the `eHerkenningAssertionConsumerServiceMockView` relies on `get_login_url` in `handle_error` (https://github.com/maykinmedia/django-digid-eherkenning/blob/master/digid_eherkenning/views/eherkenning.py#L91), I would have to override that method as well, which seems a bit excessive if we're just changing a method name :/